openstack_controller/tests/conftest.py

```diff
@@ -60,8 +60,8 @@ def replace_env_vars(match):
                 'ssl_verify': False,
                 'nova_microversion': '2.93',
                 'ironic_microversion': '1.80',
-                'openstack_cloud_name': 'test_cloud',
-                'openstack_config_file_path': '/home/openstack_controller/tests/config/openstack_config_updated.yaml',
+                '#openstack_cloud_name': 'test_cloud',
+                '#openstack_config_file_path': '/home/openstack_controller/tests/config/openstack_config_updated.yaml',
```

Comment:
Stupid question: did you mean to comment the line or is it intended to have the `#` in the keys?

Reply:
Yes I meant to comment out the line- that way it's easy to test the SDK method if needed (you just need to uncomment the line instead of paste/type in the whole path and cloud name

Reply:
haha It's not commented in fact 😅 You put the `#` in the key

Reply:
Ah yes you are correct, but they are effectively commented out, since the way the check determines whether to use the SDK method is if the `openstack_config_file_path` config key is set:  https://github.com/DataDog/integrations-core/blob/master/openstack_controller/datadog_checks/openstack_controller/config.py#L58, and now it is not set since the `#openstack_config_file_path` key is set instead 😅 

```
2024-03-11 17:12:54 UTC | CORE | WARN | (pkg/collector/python/datadog_agent.go:131 in LogMessage) | openstack_controller:bd650eb81884eca9 | (base.py:495) | Detected potential typo in configuration option in openstack_controller/instances section: `#openstack_cloud_name`. Did you mean openstack_cloud_name?
2024-03-11 17:12:54 UTC | CORE | WARN | (pkg/collector/python/datadog_agent.go:131 in LogMessage) | openstack_controller:bd650eb81884eca9 | (base.py:495) | Detected potential typo in configuration option in openstack_controller/instances section: `#openstack_config_file_path`. Did you mean openstack_config_file_path?
```

I agree it's not the best/cleanest way to do it- let me know if you have any suggestions!

Reply:
nah I guess that's ok!

Reply:
Thanks!!